### DISCUSSION TITLE
Docc style markdown directives Support

### DISCUSSION THREAD

**Discussion by wildthink** [gonzalezreal/swift-markdown-ui#350] [Q&A]
I just noticed a very recent branch that looks like you are switching to use Apple's swift-markdown parser and associated API. Does this mean that Docc-like "directives" (e.g `@Intro(title: String) {...}`) are on the horizon?

Is there anything I can do to help?

**Comment by gonzalezreal** [comment]
Hi @wildthink,
If you are referring to #348, that is not switching to `swift-markdown`, but replacing the embedded `cmark-gfm` C library with the `swift-cmark` package.

**Reply by wildthink** [reply]
Ah, do you have any plans to switch over to swift-markdown? My primary interest here is the support of BlockDirectives.

**Reply by gonzalezreal** [reply]
There are no plans at the moment, but AFAIK this fork is using `swift-markdown`:
https://github.com/lovetodream/swift-markdown-ui/tree/feature/swift-markdown-refactor
